transformer_engine/pytorch/tensor/float8_tensor.py

```diff
@@ -919,6 +919,14 @@ def _reset_caches(self) -> None:
         """
         self._transpose_invalid = True
 
+    def remove_caches(self) -> None:
```

Comment:
Nit: I'd prefer if we treated this as an internal function and not a public API that can be relied on.
```suggestion
    def _remove_caches(self) -> None:
```